src/ResourceManager/GuestConfiguration/Commands.GuestConfiguration/help/Get-AzureRmGuestConfigurationPolicyReport.md

```diff
@@ -0,0 +1,328 @@
+```
+
+PolicyDisplayName : [Preview]: Audit Windows passwords must be at least 14 characters
+ConfigurationName : MinimumPasswordLength
+ComplianceStatus  : NonCompliant
+ComplianceReasons : {[AccountPolicy]MinimumPasswordLength}
+StartTime         : 9/18/2018 2:56:54 PM
+EndTime           : 9/18/2018 2:57:06 PM
+VM                : Microsoft.Azure.Commands.GuestConfiguration.Models.GuestConfigurationPolicyAssignmentReport+VMInfo
+
+Get detailed guest configuration policy report by report Id. The report Id is the LatestReportId property that can be found in the results of Get-AzureRmGuestConfigurationPolicyReport by initiativeId or Initiative name (please refer other examples)
+
+
+## PARAMETERS
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Detailed
+Name of a policy whose definition type is Initiative and category is Guest Configuration
+
+```yaml
+Type: System.Management.Automation.SwitchParameter
+Parameter Sets: InitiativeIdScope, InitiativeNameScope
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -InitiativeId
+Definition Id of a policy whose definition type is Initiative and category is Guest Configuration
+
+```yaml
+Type: System.String
+Parameter Sets: InitiativeIdScope
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -InitiativeName
+Name of a policy whose definition type is Initiative and category is Guest Configuration
+
+```yaml
+Type: System.String
+Parameter Sets: InitiativeNameScope
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -Latest
+Name of a policy whose definition type is Initiative and category is Guest Configuration
+
+```yaml
+Type: System.Management.Automation.SwitchParameter
+Parameter Sets: InitiativeIdScope, InitiativeNameScope
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -ReportId
+Report Id of a Guest Configuration policy report.
+A policy whose definition type is Initiative and category is Guest Configuration must be assigned to a scope to get reports.
```

Comment:
Discussed offline.  Switching from 'whose' to 'where'.

Reply:
done